src/schema/v2/partnerDocumentsConnection.ts

```diff
@@ -0,0 +1,143 @@
+import config from "config"
+import {
+  GraphQLFieldConfig,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import { flatten } from "lodash"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import {
+  connectionWithCursorInfo,
+  paginationResolver,
+} from "schema/v2/fields/pagination"
+import { ResolverContext } from "types/graphql"
+import { GravityIDFields } from "./object_identification"
+import { BodyAndHeaders, ResponseHeaders } from "lib/loaders"
+
+export const PartnerDocumentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "PartnerDocument",
+  fields: {
+    ...GravityIDFields,
+    title: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    filesize: {
+      type: new GraphQLNonNull(GraphQLInt),
+      resolve: ({ size }) => size,
+    },
+    size: {
+      type: new GraphQLNonNull(GraphQLInt),
+      deprecationReason: "Prefer `filesize`",
+    },
+    publicURL: {
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ uri }) => `${config.GRAVITY_API_BASE}/${uri}`,
+    },
+    publicUrl: {
+      type: new GraphQLNonNull(GraphQLString),
+      deprecationReason: "Prefer `publicURL`",
+      resolve: ({ uri }) => `${config.GRAVITY_API_BASE}/${uri}`,
+    },
+  },
+})
+interface GravityArgs {
+  document_ids?: string[]
+  offset: number
+  size: number
+  total_count: boolean
+}
+
+const FILTER_KEYS = ["documentIDs", "artistID", "showID"]
+
+export const PartnerDocumentsConnection: GraphQLFieldConfig<
+  any,
+  ResolverContext
+> = {
+  description: "Return partner documents if current user has CMS access.",
+  type: connectionWithCursorInfo({
+    name: "PartnerDocumentsConnection",
```

Comment:
I suspect this is where the double `Connection` is coming from, maybe this should be just `PartnerDocuments` and the `Connection` is added automatically?

Reply:
Updated with https://github.com/artsy/metaphysics/pull/4489/commits/c4ecc372fa614158a3cf9b1f9705c2b249c67b11!